concordia/admin.py

```diff
@@ -30,6 +30,46 @@
 from .views import ReportCampaignView
 
 
+def publish_item_action(modeladmin, request, queryset):
+    """
+    Mark all of the selected items and their related assets as published
+    """
+
+    count = queryset.filter(published=False).update(published=True)
+    asset_count = 0
+    for item in queryset:
+        asset_count += Asset.objects.filter(item=item, published=False).update(
```

Comment:
I think we could avoid the loop by making this `asset_count += Asset.objects.filter(item__in=queryset, …)`

Reply:
I was going to ask if there was a way to avoid the loop. Thanks